dashboard/app/views/levels/editors/_blockly.html.haml

```diff
@@ -1,6 +1,6 @@
 - content_for(:head) do
-  = javascript_include_tag 'js/blockly'
-  - if @level.game && @level.game.app
+  - if @level.is_a?(Blockly) && !@level.game.uses_droplet?
+    = javascript_include_tag 'js/blockly'
     = javascript_include_tag "js/en_us/blockly_locale"
     = javascript_include_tag "js/en_us/common_locale"
     = javascript_include_tag "js/en_us/#{@level.game.app}_locale"
```

Comment:
odd that that's true for `FrequencyAnalysis` but not `Vigenere`. I think that'd be worthwhile.